### PR TITLE
Personalization endpoint

### PR DESCRIPTION
**Pull request by jonathanwgoodwin**
@jhaber

### CODE REVIEW

File: BlazarBase/src/main/protobuf/github.proto [hunk 1]
```diff
@@ -37,6 +37,13 @@ message PushEvent {
     optional Sender sender = 14;
 }
 
+message PullRequestEvent {
+    optional string action = 1;
```
Comment (jhaber) [inline BlazarBase/src/main/protobuf/github.proto]: Is there an enum for this?

File: BlazarBase/src/main/protobuf/github.proto [hunk 2]
```diff
@@ -37,6 +37,13 @@ message PushEvent {
...
+    optional string number = 2;
```
Comment (jhaber) [inline BlazarBase/src/main/protobuf/github.proto]: Is this really a string?

File: BlazarBase/src/main/protobuf/github.proto [hunk 3]
```diff
@@ -88,6 +95,55 @@ message Sender {
...
 
+message PullRequest {
+    optional string url = 1;
+    optional int64 id = 2;
+    optional string html_url = 3;
+    optional string diff_url = 4;
+    optional string patch_url = 5;
+    optional string issue_url = 6;
+    optional int32 number = 7;
+    optional string state = 8;
```
Comment (jhaber) [inline BlazarBase/src/main/protobuf/github.proto]: Is this an enum?

File: BlazarBase/src/main/protobuf/github.proto (hunk 2 above)
Comment (jonathanwgoodwin) [inline BlazarBase/src/main/protobuf/github.proto]: :-1: I'll go through this again in the morning after :coffee:

File: BlazarBase/src/main/protobuf/github.proto (hunk 1 above)
Comment (jonathanwgoodwin) [inline BlazarBase/src/main/protobuf/github.proto]: I can make one sure.

File: BlazarBase/src/main/protobuf/github.proto (hunk 3 above)
Comment (jonathanwgoodwin) [inline BlazarBase/src/main/protobuf/github.proto]: For me: https://developer.github.com/v3/pulls/#parameters

**Pull request closed without merging**